src/relay_pic/nbproject/Makefile-variables.mk

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
This file is auto generated and should not be tracked. (Same with all other `Makefile-*.mk` files)